homeassistant/components/uptimerobot/sensor.py

```diff
@@ -0,0 +1,53 @@
+"""UptimeRobot sensor platform."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN
+from .entity import UptimeRobotEntity
+
+SENSORS_INFO = {
+    "0": {"state": "Pause", "icon": "mdi:television-pause"},
+    "1": {"state": "Not checked yet", "icon": "mdi:television"},
+    "2": {"state": "Up", "icon": "mdi:television-shimmer"},
```

Comment:
Since these values are static should they have translations?

Reply:
I agree, we should add backend translations here:

<https://developers.home-assistant.io/docs/internationalization/core?_highlight=backend&_highlight=local#stringssensorjson>

This would need the state to become the lower case as well.
Better add it now, instead of later (or else it would be a known breaking change down the road).

Reply:
Done and testing

Reply:
Pushed